analyzer/codechecker_analyzer/buildlog/build_manager.py

```diff
@@ -91,6 +92,16 @@ def perform_build_command(logfile, command, context, keep_link, silent=False):
             if keep_link or ('CC_LOGGER_KEEP_LINK' in log_env and
                              log_env['CC_LOGGER_KEEP_LINK'] == 'true'):
                 log_env['CC_LOGGER_KEEP_LINK'] = 'true'
+
+            is_debug = verbose and verbose in ['debug', 'debug_analyzer']
+            if is_debug and 'CC_LOGGER_DEBUG_FILE' not in log_env:
+                log_file = os.path.join(os.path.dirname(logfile),
+                                        'logger.debug')
```

Comment:
What do you think adding codechecker to the file name.
```suggestion
                                        'codechecker.logger.debug')
```
If it is not deleted for some reason it is easier to check which tool created that file.